src/deploy/functions/triggerRegionHelper.ts

```diff
@@ -0,0 +1,51 @@
+import * as backend from "./backend";
+import * as storage from "../../gcp/storage";
+import { FirebaseError } from "../../error";
+import { logger } from "../../logger";
+
+const noop = (): Promise<void> => Promise.resolve();
+
+const LOOKUP_BY_EVENT_TYPE: Record<string, (ep: backend.EventTriggered) => Promise<void>> = {
+  "google.cloud.pubsub.topic.v1.messagePublished": noop,
```

Comment:
Curious, why do we prefer a noop here instead of the if statement check that we had before?

Reply:
Having a function pointer for noop makes it clear to the dispatching code whether something is "handled" by doing nothing or if we don't know what we're supposed to do with it. Once this code gets more event types in the table it'll seem more obvious that it's cleaner to have a flat dispatch table instead of a file full of `if/else`